astropy/modeling/core.py

```diff
@@ -2315,6 +2409,95 @@ def _custom_model_wrapper(func, fit_deriv=None):
     return type(model_name, (FittableModel,), members)
 
 
+def render_model(model, arr=None, coords=None):
+    """
+    Evaluates a model on an input array. Evaluation is limited to
+    a bounding box if the `Model.bounding_box` attribute is set.
+
+    Parameters
+    ----------
+    model : `Model`
+        Model to be evaluated.
+    arr : `numpy.ndarray`, optional
+        Array on which the model is evaluated.
```

Comment:
For congruence with methods in Numpy and Scipy it would be nice if there were just called `out`.